packages/server/lib/config.js

```diff
@@ -65,6 +65,7 @@ hosts                           screenshotsFolder
 numTestsKeptInMemory            supportFile
 port                            supportFolder
 projectId                       videosFolder
+quiet
```

Comment:
re: the changes in this file, since it's a CLI argument/module option, it doesn't need to be added to `config`

Reply:
Done